CHANGELOG.md

```diff
@@ -9,11 +9,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Headless Firefox
+<<<<<<< HEAD
 - Auto-expansion on selected subjects on the Discover page
 - `noscript` message if JavaScript is disabled
 - Favicons are updated to reflect their provider, on a branded domain or on a /preprints/provider page
 - DOI message, used on the preprint detail page to show users when they will have a DOI for their preprint.
 - Show a message on detail page when a DOI is created but not yet minted that it takes up to 24 hours to be minted.
+- Original Publication Date is added to preprint.
+  - `validated-input` component is moved to ember-osf repo
```

Comment:
Please remove the space before the `-` to make it line up with the ones on top

Reply:
The second line was indented because it was part of the change described in the first one. I don't want to put it as two separate items because the second line is actually a removal instead of an addition. But moving it to "Removed" makes it look like two unrelated things.

Reply:
I think putting this in the "Removed" section makes sense since moving `validated-input` to ember-osf wasn't really a dependency, just something else you did in this ticket. They are related, but not in a way that is probably that important to the reader of the changelog. More important is the fact that something was removed from this app.